databases/week1/Homework-week-1.sql

```diff
@@ -0,0 +1,42 @@
+-- 1. Find out how many tasks are in the task table
+SELECT COUNT(*) FROM task;
+
+-- 2. Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(*) FROM task
+WHERE due_date IS null;
+
+-- 3. Find all the tasks that are marked as done
```

Comment:
The query is correct, but as a good practice try to avoid nested queries.
A better option in this case would have been to use a join:
```
SELECT COUNT(*)
FROM task JOIN status ON status.id = task.status_id
WHERE status.name = "Done";
```